CHANGELOG.md

```diff
@@ -20,7 +25,7 @@
 - Automatically update publish state for a deployment if deployment source in vscode has changed
 - Clean up context menu to only show relevant actions [issue 203 / 204 / 206]
 - Red Hat Container Development Kit (CDK), found at https://developers.redhat.com/products/cdk, is not currently supported, but is planned. Attempts to create a CDK server adapter will fail. 
-- Configuration option added to choose which JDK to use when starting the RSP server itself. See https://github.com/redhat-developer/vscode-server-connector/pull/229 to learn more.
+- Configuration option added to choose which JDK to use when starting the RSP server itself. See https://github.com/redhat-developer/vscode-adapters/pull/229 to learn more.
```

Comment:
vscode-server-connector is the new repo name, why are you staying on the old name ?